Sprintfinity3902/Dungeon/GameState/LoseWrapper.cs

```diff
@@ -54,7 +54,7 @@ public override void Update(GameTime gameTime)
 
             switch (state) {
                 case animation_state.DELAY:
-                    if (this_count > 800) {
+                    if (this_count > 700) {
```

Comment:
Could this magic number be taken care of? Currently I don't really know what 700 accomplishes.